runtime/server/controller.go

```diff
@@ -320,12 +320,21 @@ func (s *Server) applySecurityPolicyMetricsView(ctx context.Context, instID stri
 
 // applySecurityPolicyIncludesAndExcludes rewrites a metrics view based on the include/exclude conditions of a security policy.
 func (s *Server) applySecurityPolicyMetricsViewIncludesAndExcludes(mv *runtimev1.MetricsViewV2, policy *runtime.ResolvedMetricsViewSecurity) (*runtimev1.MetricsViewV2, bool) {
-	if policy == nil || (len(policy.Include) == 0 && len(policy.Exclude) == 0) {
+	if policy == nil {
 		return mv, false
 	}
-
 	mv = proto.Clone(mv).(*runtimev1.MetricsViewV2)
 
+	if policy.ExcludeAll {
```

Comment:
Instead of this flag, wouldn't it work if we just check `if len(policy.Include) == 0 && len(policy.Exclude) == 0 ` here

Reply:
We wouldnt want to exclude everything if there were some exclude clauses but none of them evaluated to true.

Reply:
Adding a `ExcludeAll` felt more explicit and controlled by the security module.

Reply:
ok got it

Reply:
lets add a test case in follow up because the current test cases passes with this condition as well